modules/fontconfig/meta.nix

```diff
@@ -0,0 +1,15 @@
+{ lib, ... }:
+{
+  name = "Fontconfig";
+  homepage = "https://fontconfig.org";
+  maintainers = [ lib.maintainers.mightyiam ];
+  description = ''
+    This module is currently suffering from an inconsistency between platforms:
+
+    - On NixOS it
+      - does _not_ enable Fontconfig
+      - adds the Stylix fonts with default ordering to `fonts.fontconfig.defaultFonts`
+    - On home-manager it
+      - enables Fontconfig
+  '';
```

Comment:
I intend to fix this in a subsequent PR.

Reply:
Resolving because approved https://github.com/nix-community/stylix/pull/1381#pullrequestreview-2868937842